docs/dev/plugins/target-artifact-conventions/target-artifact-conventions.md

```diff
@@ -37,6 +37,8 @@ The purpose of this document is to specify conventions for the target artifact t
 As mentioned before our target artifact folder has a `scripts` folder.
 To prevent random names scripts also have to follow conventions.
 
+It is allowed to put some (important) scripts into the `output/` directory instead of the `output/scripts/` directory. This currently gets done for deployment scripts (`deploy.sh`)
```

Comment:
For the CloudFormation plugin, the `deploy.sh` still gets put in the `output/scripts/` directory, should this be changed?

Reply:
I also thought that scripts go into the `output/scripts/` directory.

Reply:
this was "problem" from my side, did fix it at its origin. This part has also been removed